library/network/src/modules/SuSEFirewall.rb

```diff
@@ -850,6 +850,20 @@ def Import(import_settings)
       nil
     end
 
+    def _sf2_to_firewalld_zone(zone)
+      sf2_to_firewalld_map = {
+        "INT" => "trusted",
+        "EXT" => "external",
+        "DMA" => "dmz"
+      }
+
+      if sf2_to_firewalld_map.has_key?(zone)
+        sf2_to_firewalld_map[zone]
+      else
+        zone
+      end
```

Comment:
This can be written as `sf2_to_firewalld_map.fetch(zone, zone)`. http://ruby-doc.org/core-2.2.0/Hash.html#method-i-fetch
Or even  `sf2_to_firewalld_map[zone] || zone` if we know that the values cannot be Booleans.